src/sidebar/util/test/groups-test.js

```diff
@@ -1,8 +1,15 @@
-import { combineGroups } from '../groups';
+import { combineGroups, $imports } from '../groups';
 
 describe('sidebar.util.groups', () => {
+  let fakeServiceConfig;
   describe('combineGroups', () => {
-    it('labels groups in both lists as isMember true', () => {
+    beforeEach(() => {
+      fakeServiceConfig = sinon.stub().returns(null);
+      $imports.$mock({
+        '../service-config': fakeServiceConfig,
+      });
+    });
+    it('labels groups in both lists as `isMember` true', () => {
```

Comment:
Can you add a line break above the first test here, please, to aid readability?

Reply:
oops! my bad